src/components/Comment/index.jsx

```diff
@@ -2,25 +2,73 @@ import { useState } from "react";
 import comments from "../../data/comments";
 import CommentElement from "./CommentElement";
 
-const Comment = () => {
-	// TODO 1: 가짜 comments 불러와서 관리해야겟즤
-
-	// TODO 2: comment추가하는 input 관리해줘야겟지
-
-	// TODO 3: comment Form 제출됐을때 실행되는 함수 만들어줘
-
-	// TODO 4: commet Delete 하는 함수 만들어죠
-	return (
-		<div className="w-full mt-5 self-start">
+const Comment = ({ postId }) => {
+  // TODO 1: 가짜 comments 불러와서 관리해야겟즤
+  const [commentList, setCommentList] = useState(
+    comments.filter((comment) => comment.post === parseInt(postId))
+  );
+  // TODO 2: comment추가하는 input 관리해줘야겟지
+  const [commentData, setCommentData] = useState({
+    id: commentList.length + 1,
+    content: "",
+    created_at: new Date(),
+    post: 1,
+    author: {
+      id: 1,
+      username: "user1",
+    },
+  });
+  const handleCommentData = (e) => {
+    const { id, value } = e.target;
+    setCommentData({ ...commentData, [id]: value });
+  };
+  // TODO 3: comment Form 제출됐을때 실행되는 함수 만들어줘
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    setCommentList((commentList) => [...commentList, commentData]);
+    setCommentData({
+      id: commentList.length + 1,
+      content: "",
+      created_at: new Date(),
+      post: 1,
+      author: {
+        id: 1,
+        username: "user1",
+      },
+    });
+  };
+  // TODO 4: commet Delete 하는 함수 만들어죠\
+  const deleteComment = (e) => {
+    const { id } = e;
```

Comment:
```suggestion
  const deleteComment = (comment) => {
    const { id } = comment;
```
통상적으로 e라고 적으면 event입니다. commetElement에서 deleteComment를 호출할때를 보면 인자로 comment를 넣어주고 있습니다. 여기서는 comment라고 적어주는게 혼란의 여지를 없애줄거에요!